test/integration/targets/ec2_vpc_route_table/tasks/main.yml

```diff
@@ -0,0 +1,471 @@
+- block:
+
+  - name: create VPC
+    ec2_vpc_net:
+      cidr_block: 10.228.228.0/22
+      name: "{{ resource_prefix }}_vpc"
+      state: present
+      region: "{{ aws_region }}"
+      aws_access_key: "{{ aws_access_key|default(omit) }}"
```

Comment:
Can you set these as environment vars on the block, please? To test insufficient credentials, make another block.
Example:
https://github.com/ansible/ansible/pull/31892/files#diff-b7e5b439f984647e05df1d4531f0b626R69

Reply:
Is there a reason this is preferred over task variables? Is the preference documented anywhere?

Reply:
I'll go with YAML as per IRC